buildscripts/tests/test_update_test_lifecycle.py

```diff
@@ -1009,6 +1009,15 @@ def test_clean_up_description_empty(self):
         expected = "_None_"
         self.assertEqual(expected, desc)
 
+    def test_truncate_description(self):
+        desc = "a" * (JiraIssueCreator._MAX_DESCRIPTION_SIZE - 1)
+        self.assertTrue(desc == JiraIssueCreator._truncate_description(desc))
+       
+        desc += "a"
+        self.assertTrue(desc == JiraIssueCreator._truncate_description(desc))
+
+        desc += "a"
+        self.assertTrue(len(desc) <= len(JiraIssueCreator._truncate_description(desc)))
```

Comment:
This expression will be `False`.
I believe what you want to check is that `len(update_test_lifecycle.JiraIssueCreator._truncate_description(desc))` is inferior or equal to `update_test_lifecycle.JiraIssueCreator._MAX_DESCRIPTION_SIZE)`.